### PR TITLE
Apply our own response specification to our own api 

### PR DESCRIPTION
**Pull request by stivni**

### CODE REVIEW

File: src/general.apib [hunk 1]
```diff
@@ -14,7 +14,8 @@ Get a list of all users
                 + email: `john@teamleader.eu` (string)
                 + gsm: `+32412654321` (string)
                 + phone: `+3291239843` (string)
-                + team_id: `1` (number)
+                + team: (object)
```
Comment (stivni) [inline src/general.apib]: but should we use `teams : (array)` for the future?

File: src/companies.apib [hunk 2]
```diff
@@ -20,7 +20,26 @@ Search companies
...
+                + invoice_email: `invoicing@example.com` (string, optional)
+                + phone: `092980615` (string)
+                + fax: `091234567` (string)
+                + website: `https://piedpiper.com` (string)
+                + address (Address)
+                + iban: `BE12123412341234` (string)
+                + bic: `BICBANK` (string)
+                + language: `NL` (string)
+                + tags (object)
+                    + _items : tag1, tag2 (array[string])
```
Comment (stivni) [inline src/companies.apib]: Q : would it make sense for out tags to be objects with only a name too?
In that case, we could later include on _actions_ for it... (the untag action for it, for example)

**Comment by vanbrabantf** [comment]
Q: doesn't our spec go much further then that?
I know we didn't want to implement it all at once, but wouldn't a selected rollout just cause more confusion?

**Comment by stivni** [comment]
@vanbrabantf, yes, it should include types, links, embedded data, included data, ...
but I think we have more important things to tackle first... like relaunch a working stable api

File: src/companies.apib (hunk 2 above)
Comment (stivni) [inline src/companies.apib]: @teamleadercrm/developers any opinions?

Comment (regularjack) [inline src/companies.apib]: It makes sense but I would only implement it when needed

**Pull request merged**